ruby_activerecord/db/migrate/20191208162041_create_questions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateQuestions < ActiveRecord::Migration[5.1]
+  def up
+    create_table :questions do |t|
+      t.string :comment
```

Comment:
best practice is to add DB-level validation, if a field should never be empty
Do something like this
```ruby
t.string :username, null: false, default: 'Anon'
```
and you will no longer need a method `ensure_username`